scripts/copydocs.py

```diff
@@ -70,11 +55,12 @@ def f(k):
 
         return {k: f(k) for k in keys}
 
-    def save_many(self, docs, comment=None):
+    def save_many(self, docs: list[dict | web.storage], comment: str = None) -> None:
         """
 
         :param typing.List[dict or web.storage] docs:
         :param str or None comment: only here to match the signature of OpenLibrary api
+        :return None:
```

Comment:
```suggestion
```

Reply:
Line 61 is OK to keep because line 62 adds useful information to the annotations.